featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -1052,6 +1053,18 @@ def gen_feat_names(self):
             assert base_feature.hash() != join_time_split.hash()
 
 
+def test_tranform_stack_agg(es):
+    topn = ft.Feature(es['log']['product_id'],
+                      parent_entity=es['customers'],
+                      primitive=NMostCommon(n=3))
+    with pytest.raises(AssertionError):
+        ft.Feature(topn, primitive=Percentile)
+    ft.dfs(entityset=es,
+           target_entity="log",
+           agg_primitives=[NMostCommon],
+           trans_primitives=[Percentile])
```

Comment:
can you move this dfs call to a separate test case in `test_deep_feature_synthesis` and add an assert statement to confirm no Percentile(NMostCommon()) features we created?